util/threadpool_imp.cc

```diff
@@ -202,10 +202,12 @@ void ThreadPoolImpl::Impl::BGThread(size_t thread_id) {
        }
     }
 
-    if (IsLastExcessiveThread(thread_id)) {
+    if (!exit_all_threads_ && IsLastExcessiveThread(thread_id)) {
```

Comment:
Can this be an `else if (IsLastExcessiveThread())` branch together with the preceding `if (exit_all_threads_)`?

Reply:
Yes, done.